requirements/requirements-testing.txt

```diff
@@ -0,0 +1,7 @@
+docker==5.0.3
+mock==3.0.5
+model-bakery==1.5.*
+pyspark==3.2.1
+pytest-cov==2.10.1
+pytest-django==4.2.0
+pytest==6.2.*
```

Comment:
Typically a `-dev` or `[dev]` set of requirements is what is used for environments that run tests, like continuous integration environments and local development environments. 

We have that with `requirements-dev.txt`, which should have most or all of these dependencies and then maybe a few others.

I think the intention of the `usaspending-ci` docker compose service (container) was to allow tests to run locally

Reply:
My original thought was to use the existing `requirements-dev.txt` file, but it would install a few extra non-testing libraries into the testing image. I realize that this current approach would mean keeping to requirement files up-to-date if we were to change the pinned version of a library and that would add some unnecessary maintenance overhead.

I'm perfectly happy to use `requirements-dev.txt` instead of creating a new requirements file as long as there are no objects to a few extra non-testing packages being added to the testing image?